subsys/bluetooth/audio/micp_mic_ctlr.c

```diff
@@ -515,6 +516,47 @@ int bt_micp_mic_ctlr_conn_get(const struct bt_micp_mic_ctlr *mic_ctlr, struct bt
 	return 0;
 }
 
+int bt_micp_mic_ctlr_chrc_get(const struct bt_micp_mic_ctlr *mic_ctlr, uint16_t *mute_handle)
+{
+	CHECKIF(mic_ctlr == NULL)
+	{
+		LOG_DBG("NULL mic_ctlr pointer");
+		return -EINVAL;
+	}
+
+	if (mic_ctlr->conn == NULL) {
+		LOG_DBG("mic_ctlr pointer not associated with a connection. "
+			"Do discovery first");
+		return -ENOTCONN;
+	}
+
+	*mute_handle = mic_ctlr->mute_handle;
+
+	return 0;
+}
+
+int bt_micp_mic_ctlr_aics_chrc_get(const struct bt_micp_included micp_included,
+				   uint16_t *state_handle, uint16_t *gain_handle,
```

Comment:
I'd consider some struct to gather those instead of so many output parameters, but since this is testing API I'll not be very strict about this :)